data/human/dirt belt jobs.txt

```diff
@@ -138,6 +138,42 @@ mission "To Remembrance Day celebration [2]"
 		dialog `Spaceport workers already festively dressed for Remembrance Day remove the cargo of <commodity> from your ship and hand you your payment of <payment>.`
 
 
+mission "To Remembrance Day celebration [3]"
+	name `Remembrance Day celebration`
+	job
+	repeat
+	description `Bring <fare> to <destination> by <date> for the Remembrance Day celebration on February 19th. Payment is <payment>.`
+	passengers 20 4 .15
+	deadline
+	to offer
+		random < ( "passenger space" - 30 ) * 2
+		or
+			and
+				random < 20
+				month == 12
+				day >= 21
+			and
+				random < 90
+				month == 1
+			and
+				random < 30
+				month == 2
+				day < 19
+	to fail
+		month == 2
+		day == 19
```

Comment:
These Remembrance Day jobs should have their failure condition match how the Earth Day jobs do it.
```suggestion
		month * 100 + day >= 219
```